### ISSUE TITLE
Long text for boolean does not format well

### ISSUE THREAD

**Issue by MikaelSmith** [replicatedhq/kots#611]
<img width="1299" alt="Screen Shot 2020-04-21 at 10 44 38 AM" src="https://user-images.githubusercontent.com/8162496/79896900-ce372700-83bd-11ea-9193-a82d7414871f.png">
doesn't look great. This is generated by
```
    - name: ha_object_store
      title: Highly Available Object Store
      help_text: |
        This application includes MinIO as an in-cluster Object Store. You can enable a highly
        available deployment mode to avoid downtime during upgrades or spent recovering if a host
        node becomes unavailable.
      type: bool
      default: "0"
```

**Comment by salahalsaleh** [comment]
@MikaelSmith This issue should be addressed in the new version of the config renderer https://github.com/replicatedhq/kots/pull/609

**Comment by MikaelSmith** [comment]
Cool, looking forward to all the 1.16 updates.

**Comment by marccampbell** [comment]
Closing as it was fixed in 1.16.0
